aws-logs-logstream/src/main/java/software/amazon/logs/logstream/Translator.java

```diff
@@ -0,0 +1,121 @@
+package software.amazon.logs.logstream;
+
+import software.amazon.awssdk.awscore.exception.AwsServiceException;
+import software.amazon.awssdk.services.cloudwatchlogs.model.*;
+
+import software.amazon.awssdk.services.cloudwatchlogs.model.ResourceNotFoundException;
+import software.amazon.cloudformation.exceptions.*;
+import software.amazon.cloudformation.proxy.*;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.HashSet;
+import java.util.HashMap;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * This class is a centralized placeholder for
+ *  - api request construction
+ *  - object translation to/from aws sdk
+ *  - resource model construction for read/list handlers
+ */
+
+public class Translator {
+
+  public static BaseHandlerException translateException(final AwsServiceException e) {
```

Comment:
If this method is only used in testing probably move this function to test package.

Reply:
function also used in BaseHandlerStd